MIGRATION_SUMMARY.md

```
@@ -0,0 +1,271 @@
+# ERD 기반 Conversation 구조 변경 완료
+
+## 📋 변경 사항 요약
+
+### 기존 구조 → 새로운 ERD 구조
+
+#### 엔티티 변경
+| 기존 | 새로운 | 주요 변경사항 |
```

markdownlint-cli2 (0.20.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)